docs/src/validator/best-practices/operations.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Validator Operations Best Practices
+sidebar_label: General Operations
+---
+
+After you have successfully gotten a [validator running on testnet](../get-started/setup-a-validator.md) (or another cluster of your choice), you will want to become familiar with how to operate your validator on a day-to-day basis. During daily operations, you will be [monitoring your server](./monitoring.md), updating software regularly (both the Solana validator software and operating system packages), and managing your stake account and identity account.
+
+All of these skills are critical to practice. Maximizing your validator uptime is an important part of being a good operator.
+
+## Educational Workshops
+
+The Solana validator community holds regular educational workshops. You can watch past workshops through the [solana validator educational workshops playlist](https://www.youtube.com/watch?v=86zySQ5vGW8&list=PLilwLeBwGuK6jKrmn7KOkxRxS9tvbRa5p).
+
+## Help with the validator command line
+
+From within the Solana CLI, you can execute the `solana-validator` command with the `--help` flag to get a better understanding of the flags and sub commands available.
+
+```
+solana-validator --help
+```
+
+## Restarting your validator
+
+There are many operational reasons you may want to restart your validator. As a best practice, you should avoid a restart during a leader slot. A [leader slot](../../terminology.md#leader-schedule) is the time when your validator is expected to produce blocks. For the health of the cluster and also for your validator's ability to earn transaction fee rewards, you do not want your validator to be offline during an opportunity to produce blocks.
+
+To see the full leader schedule for an epoch, use the following command:
+
+```
+solana leader-schedule
+```
+
+Based on the current slot and the leader schedule, you and calculate open time windows where your validator is not expected to produce blocks.
+
+Assuming you are ready to restart, you may use the `solana-validator exit` command.  The command exits your validator process when an appropriate idle time window is reached.  Assuming that you have systemd implemented for your validator process, the validator should restart automatically after the exit.  See the below help command for details:
+
+```
+solana-validator exit --help
+```
+
+## Upgrading
+
+There are many ways to upgrade the [Solana software](../../cli/install-solana-cli-tools.md). As an operator, you will need to upgrade often, so it is important to get comfortable with this process.
+
+> **Note** validator nodes do not need to be offline while the newest version is being downloaded or built from source.  All methods below can be done before the validator process is restarted.
```

Comment:
Might be worth clarifying that a new version can be installed before the exit command is needed... perhaps giving an example?

```
1. solana-install init 1.14.17
   OR
   sh -c "$(curl -sSfL https://release.solana.com/v1.14.17/install)"

2. solana-validator --ledger /mnt/ledger exit --max-delinquent-stake 10 --monitor
```